library/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/recorder/mapper/MaskAllTextWireframeMapper.kt

```diff
@@ -13,10 +13,22 @@ internal class MaskAllTextWireframeMapper(
 ) : TextWireframeMapper(viewWireframeMapper) {
 
     override fun resolveTextValue(textView: TextView): String {
-        return String(CharArray(textView.text.length) { CHARACTER_MASK })
+        val text = textView.text
+        return String(
+            CharArray(text.length) {
+                val character = text[it]
+                if (character == CHARACTER_SPACE || character == CHARACTER_NEW_LINE) {
```

Comment:
you can use `character.isWhitespace()` to check this rather than comparing to our own hardcoded values